code/modules/html_interface/voting/voting.dm

```diff
@@ -158,6 +167,7 @@ var/global/datum/controller/vote/vote = new()
 	return .
 
 /datum/controller/vote/proc/announce_result()
+	stack_trace("Fuck my shit up. Lock is \[[lock]]")
```

Comment:
I'm trying to debug the voting controller because I can't reproduce the bug locally and my first idea didn't work.